internal/device/darwin.go

```diff
@@ -25,14 +27,33 @@ func GetPower() string {
 	return out.String()
 }
 
-func Notify() {
+func CheckPower() {
 	regex := regexp.MustCompile(POWER_REGEX)
 
 	powerBytes := regex.Find([]byte(GetPower()))
 
-	power := fmt.Sprintf("%s is remaining", powerBytes)
+	power := fmt.Sprintf("%s", powerBytes)
 
-	powerCmd := fmt.Sprintf(`display notification %q with title "Power"`, power)
+	Notify(power)
+}
+
+func PowerStats(statsStr string) (string, string) {
```

Comment:
`(string, string)` doesn't tell the reader much, does it? Could we name these result parameters to help indicate what they signify? For example:

```suggestion
func PowerStats(statsStr string) (percent, status string) {
```

Reply:
Though since `percent` is a ratio, perhaps `float64` would be better here.

Reply:
Interesting point.  Here the values are  integers  like 70 and not float values  

So,  should we decide the  type based on its value or based on the context like here its a ratio and hence float64?

Reply:
Whichever is most convenient and logical in the code that calls this function, I would say!